frontend/src/components/Forms/UpdatePermitteeForm.js

```diff
@@ -30,7 +28,7 @@ const permitteeOptions = (permit) => {
   permit.map((obj) => {
     const data = {
       value: obj.permittee[0].permittee_guid.concat(', ', obj.permit_guid),
-      option: obj.permittee[0].party.name.concat(", ", obj.permit_no)
+      label: obj.permittee[0].party.name.concat(", ", obj.permit_no)
```

Comment:
Destructure `obj` and set label to `${permittee.party.name},${permit_no}`

Thoughts?

Also, do we need to validate that `.permit_no` or `.name` will be present before accessing them, or are they always required?